### PR TITLE
Change 003

### PR DESCRIPTION
**Pull request by asaxsma**
Hey Guys - Added the MikeL's notification message for white lights and buzzer mute and Nate's linear acceleration and angular velocity messages all as part of [change_003](https://www.dropbox.com/s/aeb6gcugjd70319/User%20CAN%20Protocol%20Changelog.docx?dl=0).  This includes the amendments to the IDs.  Nate, Josh is out of town, so I was hoping that you could approve this for us.  Thanks!

### CODE REVIEW

File: as_pacmod.dbc [hunk 1]
```diff
@@ -514,6 +514,32 @@ BO_ 1030 STEERING_MOTOR_RPT_3: 8 PACMOD
...
+ SG_ VERTICAL_VALID : 5|1@0+ (1,0) [0|1] ""  CUSTOMER_ECU 
+ SG_ LATERAL_ACCEL : 8|16@0- (0.01,0) [-327.68|327.67] "m/s^2"  CUSTOMER_ECU
+ SG_ LONGITUDNAL_ACCEL : 24|16@0- (0.01,0) [-327.68|327.67] "m/s^2"  CUSTOMER_ECU
+ SG_ VERTICAL_ACCEL : 40|16@0- (0.01,0) [-327.68|327.67] "m/s^2"  CUSTOMER_ECU
+
+BO_ 1050 ANG_VEL_RPT: 8 PACMOD
+ SG_ PITCH_NEW_DATA_RX : 0|1@0+ (1,0) [0|1] ""  CUSTOMER_ECU 
+ SG_ ROLL_NEW_DATA_RX : 1|1@0+ (1,0) [0|1] ""  CUSTOMER_ECU 
+ SG_ YAW_NEW_DATA_RX : 2|1@0+ (1,0) [0|1] ""  CUSTOMER_ECU 
+ SG_ PITCH_VALID : 3|1@0+ (1,0) [0|1] ""  CUSTOMER_ECU
```
Comment (srustanAStuff) [inline as_pacmod.dbc]: units of rad or deg?

File: as_pacmod.dbc [hunk 2]
```diff
@@ -514,6 +514,32 @@ BO_ 1030 STEERING_MOTOR_RPT_3: 8 PACMOD
...
+ SG_ UNDERDASH_LIGHTS_WHITE : 1|1@0+ (1,0) [0|1] ""  PACMOD
+
+BO_ 1049 LINEAR_ACCEL_RPT: 8 PACMOD
+ SG_ LATERAL_NEW_DATA_RX : 0|1@0+ (1,0) [0|1] ""  CUSTOMER_ECU 
+ SG_ LONGITUDNAL_NEW_DATA_RX : 1|1@0+ (1,0) [0|1] ""  CUSTOMER_ECU 
+ SG_ VERTICAL_NEW_DATA_RX : 2|1@0+ (1,0) [0|1] ""  CUSTOMER_ECU 
+ SG_ LATERAL_VALID : 3|1@0+ (1,0) [0|1] ""  CUSTOMER_ECU 
+ SG_ LONGITUDNAL_VALID : 4|1@0+ (1,0) [0|1] ""  CUSTOMER_ECU 
+ SG_ VERTICAL_VALID : 5|1@0+ (1,0) [0|1] ""  CUSTOMER_ECU 
+ SG_ LATERAL_ACCEL : 8|16@0- (0.01,0) [-327.68|327.67] "m/s^2"  CUSTOMER_ECU
```
Comment (srustanAStuff) [inline as_pacmod.dbc]: The "engineering unit" values of these ranges are non-sensical.  Did we just want to convey that these are signed int16 values?

**Review by srustanAStuff — COMMENTED** [review]
.

File: as_pacmod.dbc (hunk 2 above)
Comment (asaxsma) [reply as_pacmod.dbc]: These are what we agreed to when we were all sitting around the table.

**Review by srustanAStuff — DISMISSED** [review]

**Review by driscoll85 — DISMISSED** [review]

**Comment by asaxsma** [comment]
Sorry guys, noticed indexes were off.

**Review by samrustan — DISMISSED** [review]

**Review by driscoll85 — DISMISSED** [review]

**Review by srustanAStuff — DISMISSED** [review]

**Review by JWhitleyWork — APPROVED** [review]

**Review by driscoll85 — APPROVED** [review]

**Review by srustanAStuff — APPROVED** [review]

**Comment by asaxsma** [comment]
This includes change_005.

**Pull request merged**